domains/linkliang.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "my personal homepage",
+    "repo": "https://github.com/LinkLeong",
+    "owner": {
+        "username": "LinkLeong",
+        "email": "a624669980@163.com"
+    },
+    "record": {
+        "CNAME": "LinkLeong.github.io"
```

cannot access website, 404 not found
if your records are correct then provide a preview/link of your website
```suggestion
        "CNAME": "linkleong.github.io"
```